### PR TITLE
ports/rp2/boards: Added support for YD-RP2040 by VCC-GND studio 4 8 16 MB flash which is a pin compatible PICO clone.

### PR DESCRIPTION
**Pull request by mhidro**
ports/rp2/boards: Added support for YD-RP2040 by VCC-GND studio 4 8 16 MB flash which is a pin compatible PICO clone.

VCC-GND studios have pin compatible clones of the Pico with various flash sizes. I needed a uf2 with 8mb flash. These boards also have an addressable RGB led [WS2812 ](http://152.32.187.208:8080/yd-data/YD-RP2040/YD-2040-PIN.png) on GPIO23. 

I mostly copy pasted from the we-act board and replaced strings, the resulting build uf2 seems to work perfectly for me now.

### CODE REVIEW

**Comment by rexackermann** [comment]
Hey what happened to it ?

**Comment by mhidro** [comment]
> Hey what happened to it ? Where can I get the build you made ?

https://github.com/mhidro/micropython-vcc-gnd-studio-YD-RP2040/tree/master

Although you should just use the build from one of the other RP clone boards with more ram. It was compatible when I tested it.

File: ports/rp2/boards/VCC_GND_YD_RP2040/deploy.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+### Flashing via UF2 bootloader
+
+To get the board in bootloader mode ready for the firmware update, execute
+`machine.bootloader()` at the MicroPython REPL.  Alternatively, hold
+down the BOOT button while pressing reset (RST).  The uf2 file below
```
Comment (mlksahib) [inline ports/rp2/boards/VCC_GND_YD_RP2040/deploy.md:5]: File does not includes links to precompiled UF2 binaries. Please provide the links for each version (4MB, 8MB, 16 MB)

Comment (andrewleech) [reply ports/rp2/boards/VCC_GND_YD_RP2040/deploy.md:5]: This deploy file is used to generate the matching page on micropython.org/download where the links will also be generated. This file should only give instructions, not links.

Comment (mhidro) [reply ports/rp2/boards/VCC_GND_YD_RP2040/deploy.md:5]: Why are you people necroposting on this pull. Just delete this pull request.

**Comment by freedomlives** [comment]
> > Hey what happened to it ? Where can I get the build you made ?
> 
> https://github.com/mhidro/micropython-vcc-gnd-studio-YD-RP2040/tree/master
> 
> Although you should just use the build from one of the other RP clone boards with more ram. It was compatible when I tested it.

Which clone board build do you recommend? I have the 16MB version of this board.

**Comment by mhidro** [comment]
@freedomlives 
I used this one, it worked for me.

https://micropython.org/download/PIMORONI_PICOLIPO/

**Pull request closed without merging**